From 22781312f30ba9daaede480349e07ba678733433 Mon Sep 17 00:00:00 2001
From: Kai Liu <kai.liu@suse.com>
Date: Wed, 15 Jun 2022 10:41:28 +0800
Subject: [PATCH] Revert "dmaengine: idxd: set DMA_INTERRUPT cap bit"
Patch-mainline: Never, revert stable then add mainline version back
References: bsn#83
Modified-by-SEL: No

This reverts commit 76b226eaf0550c6acf9830ef732b6063bfeeb504.

Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 drivers/dma/idxd/dma.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/dma/idxd/dma.c b/drivers/dma/idxd/dma.c
index 09ad37bbd98b..834486b82373 100644
--- a/drivers/dma/idxd/dma.c
+++ b/drivers/dma/idxd/dma.c
@@ -209,7 +209,6 @@ int idxd_register_dma_device(struct idxd_device *idxd)
 	INIT_LIST_HEAD(&dma->channels);
 	dma->dev = dev;
 
-	dma_cap_set(DMA_INTERRUPT, dma->cap_mask);
 	dma_cap_set(DMA_PRIVATE, dma->cap_mask);
 	dma_cap_set(DMA_COMPLETION_NO_ORDER, dma->cap_mask);
 	dma->device_release = idxd_dma_release;
-- 
2.35.1

